### DISCUSSION TITLE
Does a 2016 RHD BMW i3 60Ah support "remote_air_conditioning_stop"?

### DISCUSSION THREAD

**Discussion by rsteininger** [bimmerconnected/bimmer_connected#413] [Q&A]
hi there, 
I finally found some time to get the "remote_air_conditioning" working via CLI. 

However, "remote_air_conditioning_stop" doesn't seem to do anything (which is probably confirmed by the "ExecutionState.ERROR" after about a minute or so) as well as that there seems no option to stop pre-conditioning from my Android App either ... 

Can anyone pls confirm if that is really the case with a 2016 (our) i3?

Also, I have not yet found any variable showing the active state in the JSON output (status -j) hat "pre-con" is ON! Was anyone luckier than I with that? 

Thanks!

**Comment by rikroe** [comment]
If your BMW app doesn't show it, pretty sure your vehicle doesn't support it.

My current guess is that the "stop" button is configured by [`executionStopPopup`](https://github.com/bimmerconnected/bimmer_connected/blob/master/test/responses/G05/vehicles_v2_bmw_0.json#L18-L20) and the current state is stored in [`climateControl`](https://github.com/bimmerconnected/bimmer_connected/blob/master/test/responses/G05/vehicles_v2_bmw_0.json#L124-L126), although I haven't seen any value except `INACTIVE` yet.
